modules/migration-installing-legacy-operator.adoc

```diff
@@ -41,21 +41,57 @@ endif::[]
 $ sudo podman login registry.redhat.io
 ----
 
-. Download the `operator.yml` file:
+. Download the `operator.yml` file for the version of {mtc-short} you are installing:
++
+[source,terminal,subs="attributes+"]
+----
+$ sudo podman cp $(sudo podman create \
+  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v<mtc-version-number>):/operator.yml ./
```

Comment:
Where is `<mtc-version-number>` defined? I'm sorry but I'm not overly familiar with adoc.

Reply:
@eriknelson 
Re: maintainability. Richard and I discussed the MTC Operator versions yesterday and agreed that we need to use variables (called doc attributes) so that the versions are updated correctly with each release. The attributes are stored in the "_attributes/common-attributes.adoc" file. This are the current attributes for MTC:
![image](https://user-images.githubusercontent.com/31725355/170514107-21525ac9-4973-4c2d-8541-0775cd5fad20.png)